### PR TITLE
add rake task to fix imported counties

### PR DESCRIPTION
**Pull request by dreamfall**
https://app.asana.com/0/1199154381249427/1201365021223258/f

### CODE REVIEW

File: lib/tasks/form_answers.rake [hunk 1]
```diff
@@ -216,4 +216,162 @@ namespace :form_answers do
...
+      "Argyll" => "Argyll and Bute",
+      "LONDON" => "Greater London",
+      "SURREY" => "Surrey",
+      "North Somerset" => "Somerset",
+      "Wigan" => "Manchester",
+      "GREATER MANCHESTER" => "Manchester",
+      "Warwickhire" => "Warwickshire",
+      "Tyrone" => "County Tyrone",
+      "lancashire" => "Lancashire",
+      "Staffordshire" => "Stafffordshire",
```
Comment (TheDancingClown) [inline lib/tasks/form_answers.rake:253]: too many 'f's

File: lib/tasks/form_answers.rake [hunk 2]
```diff
@@ -216,4 +216,162 @@ namespace :form_answers do
...
+      "Wakefield" => "West Yorkshire",
+      "Halifax" => "West Yorkshire",
+      "HAMPSHIRE" => "Hampshire",
+      "WEST SUSSEX" => "West Sussex",
+      "Herts" => "Hertfordshire",
+      "Berkshire," => "Berkshire",
+      "Durham" => "County Durham",
+      "DORSET" => "Dorset",
+      "Londonderry" => "County Londonderry",
+      "STAFFORDSHIRE" => "Stafffordshire",
```
Comment (TheDancingClown) [inline lib/tasks/form_answers.rake:269]: and here

File: lib/tasks/form_answers.rake [hunk 3]
```diff
@@ -216,4 +216,162 @@ namespace :form_answers do
...
+      "Hackney" => "Greater London",
+      "Hitchin Herts" => "Hertfordshire",
+      "Perthshire" => "Perth and Kinross",
+      "Aberdeen city" => "Aberdeenshire",
+      "Doncaster" => "South Yorkshire",
+      "Kincardineshire" => "Aberdeenshire",
+      "Blaenau Gwent" => "Monmouthshire",
+      "West yorks" => "West Yorkshire",
+      "west Yorkshire" => "West Yorkshire",
+      "Wolverhampton" => "Stafffordshire",
```
Comment (TheDancingClown) [inline lib/tasks/form_answers.rake:304]: and here

**Review by TheDancingClown — APPROVED** [review]

File: lib/tasks/form_answers.rake (hunk 1 above)
Comment (dreamfall) [reply lib/tasks/form_answers.rake]: 🔥

**Review by TheDancingClown — APPROVED** [review]

**Pull request merged**